kubernetes/namespaces/loki/loki_values.yml

```diff
@@ -0,0 +1,64 @@
+# We are deploying a replicated single binary (not components)
+deploymentMode: SingleBinary
+
+loki:
+  # Disable authentication, we are in-cluster
+  auth_enabled: false
+
+  # Set replication to 3 replicas
+  commonConfig:
+    replication_factor: 3
+
+  # We have one schemaConfig, storing data in TSDB format in S3
+  schemaConfig:
+    configs:
+      - from: 2024-04-01
+        store: tsdb
+        object_store: s3
+        schema: v13
+        index:
+          prefix: index_
+          period: 24h
+
+  # Storage type is S3 (Minio)
+  storage:
+    type: s3
+
+  # Enable volume querying endpoint (for Grafana Explore Logs)
+  limits_config:
+    volume_enabled: true
+
+singleBinary:
+  # Set replicas to 3 again
+  replicas: 3
+  # Disable any persistence in Loki, all persistence is in S3
+  persistence:
+    enabled: false
+  # Enable the pattern ingester (for Grafana Explore Logs)
+  extraArgs:
+    - "--pattern-ingester.enabled=true"
+
+  # Add a tmpfs in /var/loki for temporary data storage
+  extraVolumes:
+    - name: var-loki-tmpfs
+      emptyDir:
+        medium: Memory
+  extraVolumeMounts:
+    - name: var-loki-tmpfs
+      mountPath: /var/loki
+
+# Reduce the memory on the memcached cluster (defaults to 8GB)
+chunksCache:
+  allocatedMemory: 1024
+
+# Scale the scalable replicas to 0 since we deploy single binary
+read:
+  replicas: 0
+write:
+  replicas: 0
+backend:
+  replicas: 0
+
+# Enable Minio for S3 storage
+minio:
+  enabled: true
```

Comment:
How does this magic work? Are we deploying minio ourselves? How does loki know where it is?